doc/manual/manual.tex

```diff
@@ -8485,23 +8488,249 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+compositional field.
+We ran both on the same cluster at global refinements 5--8 using one node with four CPUs and 
+9--10 using two nodes with 16 CPUs.
+Our results are shown in Table~\ref{tab:vof-runtime-comparison-table}.
+In all of the computations shown in Table~\ref{tab:vof-runtime-comparison-table} we used a CFL 
+number of $\sigma=0.5$.
+Due to the change in the CFL number from $\sigma = 1.0$ in Table~\ref{tab:runtime-table} to 
+$\sigma = 0.5$ in Table~\ref{tab:vof-runtime-comparison-table} and the difference between HPC 
+clusters on which the computational results shown in the two tables were made, we can't make a
+direct quantitative comparison between the data in Tables~\ref{tab:runtime-table} 
+and~\ref{tab:vof-runtime-comparison-table}.
+
+However, we can compare the required run time for a VOF computation to that for a DG computation.
+We note that the use of the VOF advection algorithm significantly reduces the required computation time in all cases, frequently requiring less than half the time required by the DG computational field.
+
+We now examine the RMS velocity data shown in Figure~\ref{fig:vof-vk-1}.
+Other than for the case of 5 levels of uniform global refinement, the curves for the RMS 
+velocities for $6$, $7$, $8$, $9$ and $10$ levels of refinement in Figure~\ref{fig:vof-vk-1} 
+are nearly indistinguishable.
+
+Upon examining the solution at the final time, we note that the general structure of the 
+solution shown in Figure~\ref{fig:VOF_van_Keken-02} matches the form and the general structure 
+found in other versions of this benchmark such as the fourth image on the right in 
+Figure~\ref{fig:vk-1}.
+We also note that the differences in the shape of the interface based on a single refinement 
+as shown in Figure~\ref{fig:VOF_van_Keken-02} are minor, although still slightly visible.
+This is to be expected as refinement is a perturbation of the initial condition at a smaller 
+wave length.
+
+\begin{table}
+  \centering
+   \begin{tabular}{|c|c|c|c|}
+     \hline
+      Global Refinement & Number of Processors & VOF & DG \\
+      \hline
+\phantom{1}5 & \phantom{1}4 &\phantom{6}1.33 minutes\phantom{hours} &\phantom{1}2.57 minutes  \\
+\phantom{1}6 & \phantom{1}4 &\phantom{1}8.51 minutes\phantom{hours} & 19.5\phantom{7} minutes \\
+\phantom{1}7 & \phantom{1}4 &\phantom{1}1.15 hours\phantom{minutes} & \phantom{1}2.49 hours\phantom{tes} \\
+\phantom{1}8 & \phantom{1}4 &\phantom{1}8.53 hours\phantom{minutes} & 19.6 hours\phantom{tes} \\
+\phantom{1}9 & 16           &         16.30 hours\phantom{minutes}  & \phantom{1}2.72 days\phantom{tes} \\
+          10 & 16           & \phantom{1}5.17 days\phantom{minutess}&>6.00 days\phantom{tess} \\
+            \hline
+        \end{tabular}
+        \caption{\it Comparison of runtimes for the van Keken problem with VOF and and a DG computational field, in which the initial conditions for DG smoothed as described in section~\ref{paragraph:van-keken compositional fields} above.
```

Comment:
```suggestion
        \caption{\it Comparison of runtimes for the van Keken problem with VOF and a DG computational field, in which the initial conditions for DG smoothed are as described in section~\ref{paragraph:van-keken compositional fields} above.
```

Reply:
@class4kayaker @naliboff   I think that the second sentence here:

> ... problem with VOF and and a DG computational field, ...

the word 'computational' is meant to be 'compositional', is it not?